sg.tf

```diff
@@ -5,60 +5,74 @@
 #
 resource "aws_security_group" "default" {
   description = "security group for ${var.name} ALB"
-  name        = "${var.name}"
-  vpc_id      = "${data.aws_vpc.selected.id}"
+  name        = var.name
+  vpc_id      = data.aws_vpc.selected.id
 
-  tags = "${merge(map("Name", var.name), var.tags)}"
+  tags = merge(
+    {
+      "Name" = var.name
+    },
+    var.tags,
+  )
 }
 
 resource "aws_security_group" "secure" {
-  count       = "${length(var.secure_ports)}"
-  description = "${lookup(var.secure_ports[count.index], "security_group_description", "")}"
-  name        = "${lookup(var.secure_ports[count.index], "security_group")}"
-  vpc_id      = "${data.aws_vpc.selected.id}"
+  count = length(var.secure_ports)
+  description = lookup(
```

Comment:
Make this a one-liner please.

Reply:
Is a lookup even needed with v0.12? See line 26.

Reply:
See my comment above about consistency with the old module's behavior, and about issue #6.